src/helpers.js

```diff
@@ -68,6 +69,19 @@ export const getValidatedInput = (key, re) => {
   return value;
 }
 
+/**
+ * @param {string} key
+ * @param {RegExp} re regex to use for validation
+ * @return {string|undefined} {undefined} or throws an error if input doesn't match regex
+ */
+export const getValidatedEnvVars = (key, re) => {
```

Comment:
is `getValidatedInput` used anywhere now or can that fct be removed?

Reply:
Removed! Thanks!